src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/cas2/PersonUtilsTest.kt

```diff
@@ -0,0 +1,79 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.unit.service.cas2
+
+import io.mockk.every
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.springframework.http.HttpMethod
+import org.springframework.http.HttpStatus
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.client.ClientResult
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.factory.InmateDetailFactory
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.factory.OffenderDetailsSummaryFactory
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.model.PersonInfoResult
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.problem.NotFoundProblem
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.util.cas2.getFullInfoForPersonOrThrow
+
+class PersonUtilsTest : OffenderServiceTest() {
+
+  @Test
+  fun `throws NotFoundProblem when offender is PersonInfoResult-NotFound, status 404`() {
```

Comment:
It might help with readability to add a `@Nested` class around these two tests to show what function is under test, e.g. 
```
  @Nested
  inner class GetFullInfoForPersonOrThrow {
  ...
  }

Reply:
Incorporated this in OffenderServiceTest.